tests/security/with-api-prefix/allowlist.yaml

```diff
@@ -0,0 +1,30 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test allowlist endpoints.
+
+# ADMIN-CERT only. These tests require explicit rest api admin privileges.
```

Comment:
If you need a custom docker compose, it's possible now.

Reply:
This will require a custom opensearch.yml, along with custom certificates and a custom roles-mapping file to set up a cluster. 
The default behavior is to return 403 which is tested here. Should we add tests for Rest admin only and configure those to run with custom configuration? Can this be done in a follow-up PR?

Reply:
Sounds good for a future PR. We now have support for completely custom docker-compose where you can create a subfolder in tests and put things together for custom configurations.